<ui version="4.0" >
 <class>SkulptureStyleDialog</class>
 <widget class="QWidget" name="SkulptureStyleDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>386</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QMdiArea" name="mdiArea" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="cbAnimateProgressBars" >
        <property name="text" >
         <string>Animate Progress Bars</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAllowScrollBarSliderToCoverArrows" >
        <property name="text" >
         <string>Allow Scroll Bar Handle to Cover Arrows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbHideShortcutUnderlines" >
        <property name="text" >
         <string>Hide Shortcut Underlines</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>mdiArea</zorder>
  <zorder>cbAnimateProgressBars</zorder>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>false</bool>
  </property>
 </designerdata>
</ui>
